src/js/features/progress.js

```diff
@@ -11,7 +11,20 @@ import {secondsToTimeCode} from '../utils/time';
  *
  * This feature creates a progress bar with a slider in the control bar, and updates it based on native events.
  */
-
+	function setCurrentRailMain(t,fakeTime){ 
```

Comment:
This should be inside the `Object.assign(MediaElementPlayer.prototype` like the `setCurrentRailHandle` method